CHANGES/1084.misc

```diff
@@ -0,0 +1,2 @@
+
+Refactoring of ``pair_list`` implementation.
```

Comment:
This should reflect the impact the end-users would observe having updated to the version including the change. Also, the `-ing` form is not a good fit here because it should describe what has already changed, not what's in the process of changing.

Reply:
Please feel free to reword

Reply:
Note, the change doesn't affect any end-user's code.

Reply:
Updated

Reply:
Yes, that's why it's “misc”. But still, putting very vague generic statements is not really helpful in the change log document. Being more specific could make it useful, though.